samples/ei_ncs/src/main.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <ei_ncs.h>
+
+#include "input_data.h"
+
+static size_t frame_surplus;
+
+void result_ready_cb(int err)
+{
+	if (err) {
+		printk("Result ready callback returned error (err: %d)\n", err);
+		return;
+	}
+
+	const char *label;
+	float value;
+	float anomaly;
+
+	int error = ei_ncs_get_classification_results(&label, &value, &anomaly);
```

Comment:
you can reuse `err` here

Reply:
note that it's easy to make an error when standard name changes (see line 28)

Reply:
done